tests/ssg_test_suite/rule.py

```diff
@@ -226,7 +230,10 @@ def _test_target(self, target):
 
     def _modify_parameters(self, script, params):
         if self.scenarios_profile:
-            params['profiles'] = [self.scenarios_profile]
```

Comment:
Let's tell users to specify the actual virtual profile name, i.e. `(all)`, like that, there won't be any need to treat the string `all` in a special way.

Reply:
done